pokedex-with-bit/pods/doc.yml

```diff
@@ -1,6 +1,32 @@
-!DataURIPbIndexer
+!BinaryPbIndexer
 with:
-  index_filename: doc.gzip
+  index_filename: doc.gz
 metas:
-  name: doc_indexer  # a customized name
-  workspace: $WORKDIR
\ No newline at end of file
+  name: doc_idx
+  workspace: $WORKDIR
+requests:
+  on:
+    ControlRequest:
+      - !ControlReqDriver {}
+    SearchRequest:
+      - !KVSearchDriver
+        with:
+          granularity_range: [0, 0]
+          adjacency_range: [0, 1]
+      - !ExcludeQL
+        with:
+          granularity_range: [0, 2]
+          adjacency_range: [0, 1]
+          fields:
+            - embedding
+    IndexRequest:
+      - !LogInfoDriver {}
+      - !Buffer2URI {}
+      - !LogInfoDriver {}
```

Comment:
remember to removed all these `LogInfoDriver`

Reply:
@JamesTang-jinaai Would you please remove this?